frontend/src/metabase/css/components/header.css

```diff
@@ -54,26 +54,34 @@
 .EditHeader .Button {
     color: var(--brand-color);
     border: none;
-    text-transform: uppercase;
-    font-size: 0.75rem;
-    background-color: rgba(255,255,255,0.5);
-    font-weight: normal;
+    font-size: 0.875rem;
+    background-color: rgba(255,255,255,0.1);
+    font-weight: 700;
```

Comment:
Ideally we could use the `text-bold` class on these buttons instead of adding this declaration here.